app/controllers/qing_groups_controller.rb

```diff
@@ -66,8 +69,12 @@ def prepare_qing_group
   end
 
   def qing_group_params
+    condition_params = [:id, :ref_qing_id, :op, :value, :_destroy, option_node_ids: []]
     translation_keys = permit_translations(params[:qing_group], :group_name, :group_hint, :group_item_name)
-    params.require(:qing_group).permit([:form_id, :repeatable, :one_screen] + translation_keys)
+    params.require(:qing_group).permit(
+      %i[form_id repeatable one_screen] + translation_keys,
+       display_conditions_attributes: condition_params
```

Comment:
Are you sure this works? I would have thought you'd need something like: `(%i[form_id repeatable one_screen] + translation_keys) << {display_conditions_attributes: condition_params}`, but if it works it works!

Indentation is a little weird. I think it would be fine to just break first line on the comma and then indent one level.